src/test/ui/rfc-2361-dbg-macro/dbg-macro-expected-behavior.rs

```diff
@@ -1,4 +1,5 @@
 // run-pass
+// check-run-results
 // ignore-cloudabi no processes
 // ignore-emscripten no processes
 // ignore-sgx no processes
```

Comment:
[Turns out](https://github.com/rust-lang/rust/issues/63924) you were probably right to remove the ignores here -- "no processes" likely means "no `std::process`", or at least not enough of it for this test.